public/style.css

```
@@ -30,8 +30,12 @@ label {
 }
 
 table {
-  width: 100px;
+  width: auto;
   margin-left: calc(50vw - 130px);
+  white-space: nowrap;
+}
+td, th {
+  padding: 0px 30px 0 0 ;
 }
```

Place white space between css selector, and extra space that doesn't belong there
```suggestion
  width: auto;
  margin-left: calc(50vw - 130px);
  white-space: nowrap;
}

td, th {
  padding: 0px 30px 0 0;
}
```